docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1465,6 +1506,16 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/notifications/kafka/enabled`|false|
 |`fhirServer/notifications/kafka/topicName`|fhirNotifications|
 |`fhirServer/notifications/kafka/connectionProperties`|`{}`|
+|`fhirServer/notifications/nats/enabled`|false|
+|`fhirServer/notifications/nats/cluster`|nats-streaming|
+|`fhirServer/notifications/nats/channel`|fhirNotifications|
+|`fhirServer/notifications/nats/clientId`|fhir-server|
+|`fhirServer/notifications/nats/servers`|nats://nats-node1:4222,nats://nats-node2:4223,nats://nats-node3:4224|
+|`fhirServer/notifications/nats/useTLS`|false|
```

Comment:
just a thought to switch to true
it's a preferred security posture

Reply:
i kinda gave it a pass since its new/experimental, but i suppose you're right.  the default should be secure.

Reply:
yes, except it's considerably easier to not use TLS if someone wants to try it out.  NATS cluster TLS can be time-consuming to configure properly.